docs/cuda-miner.md

```diff
@@ -7,7 +7,7 @@ The release packages do not ship with a CUDA miner, but you can build it yoursel
 - Epoch node configuration
 
 The documentation below is tested on:
-- Epoch version 0.19.0
```

Comment:
@dincho I took the liberty of bumping this too.

Reply:
well, this will not be technically correct, because no one would test it on each release

Reply:
I assume CI tests are still tests. Code and documentation shall be corresponding to each other, so if we've missed a thingy - that would certainly be a bug. Iagree on the bumping.

Reply:
True, but the CI does not test CUDA install and deployment, neither we deploy GPU miners on out testnet (yet?).

I see this line more like disclaimer, thus my comment.